src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -481,8 +483,15 @@ export default {
 		flex: 1 0 auto;
 		padding: 0 calc(2 * var(--default-grid-baseline));
 
-		.date:last-child {
-			margin-right: var(--default-clickable-area);
+		.date {
+			&--hidden {
+				pointer-events: none;
+				opacity: 0;
+			}
+
+			&:last-child {
+				margin-right: var(--default-clickable-area);
+			}
```

Comment:
There is only one date for each message?

Reply:
We reserve a place for read marker
But that's up to different discussion